internal/test/test.go

```diff
@@ -104,3 +105,13 @@ func update(filename string, in []byte) error {
 func normalize(in []byte) []byte {
 	return bytes.Replace(in, []byte("\r\n"), []byte("\n"), -1)
 }
+
+// TempDir ensures a scratch test directory for unit testing purposes.
+func TempDir(t *testing.T) string {
+	t.Helper()
+	d, err := ioutil.TempDir("", "helm")
```

Comment:
`s/helm/hypper/`

Reply:
This is actually removed as your PR brings the same function, so I will drop this.